src/Validator/ArrayList.php

```diff
@@ -52,6 +52,18 @@ public function getDescription()
         return 'Value must be an array and ' . $this->validator->getDescription();
     }
 
+    /**
+     * Get Type
+     *
+     * Returns validator type.
+     *
+     * @return string
+     */
+    public function getType()
+    {
+        return self::TYPE_ARRAY;
```

Comment:
@kodumbeats How do you foresee this handling something like this https://github.com/appwrite/appwrite/blob/master/app/controllers/api/database.php#L174

Reply:
Discussed IRL - TYPE_ARRAY is sufficient for now to capture complex cases.